### ISSUE TITLE
Single-crop validation performance

### ISSUE THREAD

**Issue by nhynes** [CSAILVision/places365#6]
Would it be possible to release the single-crop validation performance? This would make it easier to verify models that have been converted from caffe [without writing a complicated data loader].

Thanks!

**Comment by shicai** [comment]
second this.

**Comment by metalbubble** [comment]
Sure. Here is the accuracy on the validation directly from caffe training.
VGG16:
 I0416 10:03:46.482185  2637 solver.cpp:404] 	Test net output #0: accuracy = 0.539833
I0416 10:03:46.482353  2637 solver.cpp:404] 	Test net output #1: loss = 1.73844 (* 1 = 1.73844 loss)

CaffNet:
I0407 01:58:51.494807 11471 solver.cpp:404] 	Test net output #0: accuracy = 0.51652
I0407 01:58:51.494874 11471 solver.cpp:404] 	Test net output #1: loss = 1.84729 (* 1 = 1.84729 loss)

ResNet152 finetuned:
I0601 23:04:56.067314 19006 solver.cpp:404]     Test net output #0: accuracy = 0.55
I0601 23:04:56.067584 19006 solver.cpp:404]     Test net output #1: loss = 1.76827 (* 1 = 1.76827 loss)

**Comment by shicai** [comment]
@metalbubble something strange here.
by using resnet 152, the averaged score over 10 crops is 54.74/85.08%.
how can it be 55% with only single crop?

**Comment by metalbubble** [comment]
Yes, indeed it is kind of weird to me. Maybe it is due to the fine-tuning of the resnet. I will manage to trian a resnet from scratch using torch and upload the numbers.

**Comment by hendrycks** [comment]
@metalbubble 

> I will manage to trian a resnet from scratch using torch and update the numbers.
Could you show the numbers for the ResNet152-places365 torch model you trained from scratch?
(I ported the Torch model to PyTorch and want to know if I ported it successfully.)

**Comment by metalbubble** [comment]
Here is the final state of the training log:

  | Test: [120][141/143]    Time 0.134  Data 0.000  top1  48.047 ( 44.815)  top5  13.281 ( 14.702)
  | Test: [120][142/143]    Time 0.121  Data 0.000  top1  42.578 ( 44.799)  top5  16.797 ( 14.717)
  | Test: [120][143/143]    Time 0.121  Data 0.000  top1  48.047 ( 44.822)  top5  15.234 ( 14.721)
  * Finished epoch # 120     top1:  44.822  top5:  14.721

**Comment by Tveek** [comment]
Verify your pytorch model(alexnet),so big gap with your Caffe model. Here is the precision of the your pytorch model

-  Prec@1 45.373 1-crop
- Prec@1 48.627 10-crop
